src/hapi_schema/db_funding.py

```diff
@@ -0,0 +1,92 @@
+"""Funding subcategory table and view."""
+
+from datetime import datetime
+
+from sqlalchemy import (
+    CheckConstraint,
+    DateTime,
+    ForeignKey,
+    String,
+    select,
+    text,
+)
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from hapi_schema.db_location import DBLocation
+from hapi_schema.utils.base import Base
+from hapi_schema.utils.view_params import ViewParams
+
+
+class DBFunding(Base):
+    __tablename__ = "funding"
+    __table_args__ = (
+        CheckConstraint(
+            "requirements_usd >= 0.0",
+            name="requirements_usd",
+        ),
+        CheckConstraint(
+            "funding_usd >= 0.0",
+            name="funding_usd",
+        ),
+        CheckConstraint(
+            "funding_pct >= 0.0",
+            name="funding_pct",
+        ),
+        CheckConstraint(
```

Comment:
On `develop` I've now implemented some helper functions for commonly occurring constraints such as this one